src/Security/Authentication/Core/src/LoggingExtensions.cs

```diff
@@ -25,99 +25,99 @@ internal static class LoggingExtensions
         static LoggingExtensions()
         {
             _remoteAuthenticationError = LoggerMessage.Define<string>(
-                eventId: 4,
+                eventId: new EventId(4, "RemoteAuthenticationFailed"),
```

Comment:
If we want the event name and property name to line up, maybe just use nameof and make them match?

```C#
private static readonly Action<ILogger, string, Exception> RemoteAuthenticationFailed;  

 RemoteAuthenticationFailed = LoggerMessage.Define<string>(
	            eventId: new EventId(4, nameof(RemoteAuthenticationFailed),
```

Reply:
nameof shouldn't be used for event names because they should never change in case of refactorings.

Reply:
https://github.com/aspnet/AspNetCore/pull/4495#issuecomment-445439659 @BrennanConroy  said that we are avoiding `nameof` so we avoid accidentally causing a breaking change

Reply:
Fair enuf